### PR TITLE
Use correct API for file open.

### PR DESCRIPTION
**Pull request by rebornix**
Yay! We love PRs! 🎊

Please include a description of your change & check your PR against this list, thanks:

- [x] Commit message has a short title & issue references
- [x] Commits are squashed 
- [x] It builds and tests pass (e.g `gulp tslint`)

More info can be found by clicking the "guidelines for contributing" link above.

### CODE REVIEW

File: src/cmd_line/commands/file.ts [hunk 1]
```diff
@@ -22,11 +23,10 @@ export class FileCommand extends node.CommandBase {
...
     }
 
-    execute() : void {
-        if (process.platform === 'darwin') {
-            vscode.commands.executeCommand("workbench.action.files.openFileFolder");
-        } else {
-            vscode.commands.executeCommand("workbench.actiow.files.openFile");
-        }
+    execute(): void {
+        let oriPath = vscode.window.activeTextEditor.document.uri.path;
```
Comment (johnfn) [inline src/cmd_line/commands/file.ts]: Very minor complaint, but I'm generally not a big fan of abbreviated variable names when it's hard to tell what it's abbreviating.

File: src/cmd_line/subparsers/file.ts [hunk 2]
```diff
@@ -11,12 +11,21 @@ export function parseEditFileCommandArgs(args: string): node.FileCommand {
     var scanner = new Scanner(args);
     scanner.skipWhiteSpace();
 
-    if (scanner.isAtEof) {
-        return new node.FileCommand({});
+    let name = "";
+    while (true) {
```
Comment (johnfn) [inline src/cmd_line/subparsers/file.ts]: I think you want to do `while (!scanner.isAtEof)`

File: src/cmd_line/subparsers/file.ts [hunk 3]
```diff
@@ -11,12 +11,21 @@ export function parseEditFileCommandArgs(args: string): node.FileCommand {
...
+        if (scanner.isAtEof) {
+            break;
+        }
+        let c = scanner.next();
+
+        if (/\s/.test(c)) {
+            break;
+        } else {
+            name += c;
+        }
```
Comment (johnfn) [inline src/cmd_line/subparsers/file.ts]: It actually seems to me like scanner itself should have a method called `readNextWord`, which this should go into.

File: src/cmd_line/commands/file.ts (hunk 1 above)
Comment (rebornix) [inline src/cmd_line/commands/file.ts]: Good catch!

File: src/cmd_line/subparsers/file.ts (hunk 3 above)
Comment (rebornix) [inline src/cmd_line/subparsers/file.ts]: Yeah, this method can benefit more later on.

Comment (rebornix) [inline src/cmd_line/subparsers/file.ts]: Yeah, this method can benefit more later on.

**Comment by rebornix** [comment]
Move `nextWord` to `Scanner`. This PR is about the existing file opening feature so I think we may have this PR merged for v0.1.

Besides, I'll have another PR taking care of Windows/Tabs bugs and documentation then users are aware of what they can do or do not with these commands.

**Comment by johnfn** [comment]
Excellent, this reads much nicer now.

**Pull request merged**